File: Programming_Languages/Python/Node.py
# Python program to introduce Binary Tree

# A class that represents an individual node in a
# Binary Tree
class Node:
	def __init__(self,data):
		self.left = None
		self.right = None
		self.data = data


# create root
root = Node(1)
root.left	 = Node(2)
root.right	 = Node(3)
root.left.left = Node(4)


#Traversal [PreOrder]

def preorder(root):
    if root == None:
        return
    
    print(root.key)
    #Recursive call
    preorder(root.left)
    preorder(root.right)


#Execute program 
preorder(root)

#Contributed by KmAlan

File: Programming_Languages/Javascript/Node.js
class Node{
    constructor(data){
        this.data=data;
        this.left=null;
        this.right=null;
    }
}

//Instance of Node class

root = new Node(1);
root.left	 = Node(2);
root.right	 = Node(3);
root.left.left = Node(4);

//Tree traversal[preorder]
const preOrder = (root) => {
    if(root == null){
        return;
    }
    console.log(root.data);
    //Recursive call
    preOrder(root.left);
    preOrder(root.right);
}
//Execute the program copy to your browser console
 preOrder(root);



 //Contibuted by KmAlan